src/tools/r2rdump/GCInfo.cs

```diff
@@ -33,8 +34,8 @@ private enum GcInfoHeaderFlags
 
         public struct InterruptibleRange
         {
-            public uint StartOffset { get; }
-            public uint StopOffset { get; }
+            public uint StartOffset { get; set; }
```

Comment:
Why did you make all the properties publicly writeable? I thought the idea was they get set as we read each data structure from the image and from then on these should all be immutable.

Reply:
The XmlSerializer required the properties to have a public setter. When it didn't have one, it didn't serialize those properties